tests/py3_driver_testers.txt

```diff
@@ -318,6 +318,9 @@ TIDRADIO_TD-H8-GMRS,@Sandmann34,7-Feb-2023
 TIDRADIO_TD-H8-HAM,@Sandmann34,7-Feb-2023
 TID_TD-M8,+Retevis_RT22,9-Dec-2022
 TID_TD-UV68,@Sandmann34,7-Feb-2023
+TIDRADIO_TD-H3,@songjunbiao,1-December-2023
+TIDRADIO_TD-H3-GMRS,@songjunbiao,1-December-2023
+TIDRADIO_TD-H3-HAM,@songjunbiao,1-December-2023
```

Comment:
Please use the same date format as the other several hundred lines in this file (1-Dec-2023)